client/manager.go

```diff
@@ -46,6 +46,7 @@ type Manager struct {
 	Notify               bool    `json:"notify"`
 	NotifyQuantity       int     `json:"notify_quantity"`
 	NotifyAfter          int     `json:"notify_after"`
+	UpscaleOnInitialJob  *bool   `json:"upscale_on_initial_job"`
```

Comment:
what is the asterisk doing here ? i'm not very familiar with go, but looking a few lines up there's another `bool` without it

Reply:
It's saying that the `UpscaleOnInitialJob` is a pointer to a `bool`, rather than a `bool` value directly. Looking a little further up, this is how most of this struct is declared - IIUC, the intent being that this struct is re-used across several different `Manager` "types" (in the sense of the HireFire API), and so these properties of the struct are more variable. Sometimes they're there, sometimes they're not, depending on which kind of "Manager" we're dealing with. The non-pointers are in every kind of "Manager."